diagnostic_feedback/public/js/src/student/student_quiz.js

```diff
@@ -0,0 +1,239 @@
+function StudentQuiz(runtime, element) {
+
+    var studentQuiz = this;
+    studentQuiz.startOver = false;
+    studentQuiz.movingToStep = false;
+
+    /* Javascript for Student view in LMS.*/
+
+    var common = new Common(),
+        form = $(".diagnostic-feedback #student_view_form"),
```

Comment:
All of these selectors need to be scoped to the `element`, since there can be multiple diagnostic feedback blocks on one page. e.g. `$(".diagnostic-feedback .student_view_form", element)`